src/Calculator/BcMathCalculator.php

```diff
@@ -0,0 +1,195 @@
+<?php
+namespace Money\Calculator;
+
+use Money\Calculator;
+use Money\Money;
+use Money\Number;
+
+/**
+ * Class BcMathCalculator
+ * @package Money\Calculator
+ */
+final class BcMathCalculator implements Calculator
+{
+    /**
+     * @var string
+     */
+    private $scale;
+
+    /**
+     * @param int $scale
+     */
+    public function __construct($scale = 14)
+    {
+        $this->scale = $scale;
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @return bool
+     */
+    public static function supported()
+    {
+        return extension_loaded('bcmath');
+    }
+
+    /**
+     * {@inheritdoc}
+     *
+     * @param $a
```

Comment:
I would not agree with this. The [phpDocumentor documentation](http://www.phpdoc.org/docs/latest/guides/inheritance.html) says "The inline tag {@inheritDoc} is used in a description to import the description of a parent element, even if the child element already has a description." So `{@inheritdoc}` is only relevant to the description. The current `Money` class also [contains inherited docblock with return docs](https://github.com/mathiasverraes/money/blob/nextrelease/src/Money.php#L529).